lib/dogma/rule/comment_format.ex

```diff
@@ -1,4 +1,6 @@
-defmodule Dogma.Rule.CommentFormat do
+use Dogma.RuleBuilder
+
+defrule Dogma.Rule.CommentFormat, [] do
```

Comment:
small comment but if there are no default config values maybe 
```elixir
defrule Dogma.Rule.CommentFormat do
```
would be better.